src/styles/index.scss

```diff
@@ -1,3 +1,2 @@
-body {
-  margin: 0;
-}
+@import url(../blocks/card.scss);
+@import url(../blocks/stars.scss);
```

Comment:
In SCSS, you should use @import './blocks/stars'; without the file extension and url() function.